hack/static/versions.yaml

```diff
@@ -0,0 +1,473 @@
+- AppID: centralized-grafana
+  AppName: centralized-grafana-34.9.3
+  AppVersion: 0.55.0
```

Comment:
so this would be the prometheus-operator version, I believe.
the thing that makes this significantly more complex (kube-prometheus-stack is probably the most complicated case) is that within kps there are many components -- prometheus-operator (the chart's appversion in this case), grafana, prometheus, alertmanager.. so maybe for kps we would need to parse the values file https://github.com/mesosphere/charts/blob/master/staging/kube-prometheus-stack/values.yaml to get the various component images?

e.g. alertmanager https://github.com/mesosphere/charts/blob/master/staging/kube-prometheus-stack/values.yaml#L421

You can take a look at the 2.2.1 release notes table https://docs.d2iq.com/dkp/kommander/2.2/release-notes/2.2.1/#component-updates at `Component Versions` column. looks like hopefully just kps where we have to add some special handling to get the other application versions